docs/web/examples/reverse-proxy.py

```diff
@@ -13,6 +13,6 @@
 from twisted.internet import reactor
 from twisted.web import proxy, server
 
-site = server.Site(proxy.ReverseProxyResource('www.yahoo.com', 80, ''))
+site = server.Site(proxy.ReverseProxyResource('www.asdf.com', 80, b''))
```

Comment:
Why the change to the hostname? Do we explicitly want `www.asdf.com` or should we change it to a [standard](https://tools.ietf.org/html/rfc2606) example.com hostname?

Reply:
The point is that most popular websites simply redirect any request to their HTTPS service. asdf.com was the first non-local non-specific non-HTTPS dummy website I could come up with. But indeed example.com is a much better option, thanks!